server/src/infra/entities/exif.entity.ts

```diff
@@ -43,6 +54,14 @@ export class ExifEntity {
   @Column({ type: 'float', nullable: true })
   longitude!: number | null;
 
+  @Column({
+    type: 'enum',
+    enum: ProjectionType,
+    enumName: 'projectionType',
+    default: ProjectionType.NONE,
+  })
+  projectionType!: ProjectionType;
```

Comment:
@jrasm91 What do you think about using the native enum of PG here? I think we should use `string` or `varchar` and similar to how we are using `type` in `AssetEntity`

Reply:
Changed to varchar